Detectors/CPV/reconstruction/src/RawDecoder.cxx

```diff
@@ -18,106 +18,122 @@
 using namespace o2::cpv;
 
 RawDecoder::RawDecoder(RawReaderMemory& reader) : mRawReader(reader),
-                                                  mRCUTrailer(),
                                                   mChannelsInitialized(false)
 {
 }
 
 RawErrorType_t RawDecoder::decode()
 {
 
-  auto& header = mRawReader.getRawHeader();
-  short ddl = o2::raw::RDHUtils::getFEEID(header);
+  auto& rdh = mRawReader.getRawHeader();
+  short linkID = o2::raw::RDHUtils::getLinkID(rdh);
   mDigits.clear();
+  mBCRecords.clear();
 
-  auto payloadwords = mRawReader.getPayload();
-  if (payloadwords.size() == 0) {
-    mErrors.emplace_back(ddl, 0, 0, 0, kNO_PAYLOAD); //add error
-    LOG(ERROR) << "Empty payload for DDL=" << ddl;
-    return kNO_PAYLOAD;
+  auto payloadWords = mRawReader.getPayload();
+  if (payloadWords.size() == 0) {
+    return kOK_NO_PAYLOAD;
   }
 
-  // if(readRCUTrailer()!=kOK){
-  //   LOG(ERROR) << "can not read RCU trailer for DDL " << ddl ;
-  //   return kRCU_TRAILER_ERROR;
-  // }
-
   return readChannels();
 }
 
-RawErrorType_t RawDecoder::readRCUTrailer()
-{
-  gsl::span<const char> payload(reinterpret_cast<const char*>(mRawReader.getPayload().data()), mRawReader.getPayload().size() * sizeof(uint32_t));
-  mRCUTrailer.constructFromRawPayload(payload);
-  return kOK;
-}
-
 RawErrorType_t RawDecoder::readChannels()
```

Comment:
ReadChannels()

Reply:
Hi, @kharlov 
here coding guidelines says "Regular functions follow camel case convention and start with a lower case letter: myFunction()."
https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Function_Names#Function_Names